### ISSUE TITLE
OK if we use prettier-eslint?

### ISSUE THREAD

**Issue by hampelm** [detroitledger/grantentry#8]
Don't want to add too much weight but I enjoy using [eslint + prettier](https://github.com/prettier/prettier-eslint)

It formats the javascript using prettier, then automatically applies any style rules you have in `eslint` 

There's a plugin for [visual studio code](https://github.com/prettier/prettier-vscode) and most of the major editors.

**Comment by hampelm** [comment]
Also, here's my typical .eslintrc.js: 

```
module.exports = {
  root:          true,
  parserOptions: {
    ecmaVersion:  6,
    ecmaFeatures: {
      jsx:                          true,
      experimentalObjectRestSpread: true,
    },
    sourceType: 'module',
  },
  plugins: ['react'],
  extends: ['eslint:recommended', 'plugin:react/recommended'],
  env:     {
    browser: true,
    es6:     true,
    jest:    true,
    mocha:   true,
    node:    true,
  }, 
  rules: {
    'no-console':  0,
    semi:          ['error', 'never'],
    'key-spacing': ['error', { mode: 'minimum', align: 'value' }],
  },
}
```

**Comment by bnchdrff** [comment]
addressing this in https://github.com/detroitledger/grantentry/tree/circle-bc

**Comment by hampelm** [comment]
Was this closed by #11?

**Comment by bnchdrff** [comment]
tis fixed!
